### PR TITLE
Always interact with ctx through facilitator functions

### PR DESCRIPTION
**Pull request by jackkleeman**

### CODE REVIEW

**Review by muhamadazmy — APPROVED** [review]
LGTM

File: facilitators.go
Comment (slinkydeveloper) [inline facilitators.go:1]: Not sure facilitators is the right name anymore for this file :)

File: facilitators.go [hunk 1]
```diff
@@ -2,111 +2,242 @@ package restate
...
+	return outputClient[O]{ctx.inner().Object(service, key, method, options...)}
 }
 
-// AwakeableAs helper function to treat [Awakeable] results as a particular type.
-func AwakeableAs[T any](ctx Context, options ...options.AwakeableOption) TypedAwakeable[T] {
-	return typedAwakeable[T]{ctx.Awakeable(options...)}
+// ObjectSend gets an Object send client by service name, key and method name
+func ObjectSend(ctx Context, service string, key string, method string, options ...options.ClientOption) SendClient[any] {
+	return ctx.inner().Object(service, key, method, options...)
 }
```
Comment (slinkydeveloper) [inline facilitators.go:61]: Maybe not for now, but can we have Workflow/WorkflowSend too (you might want to call workflows built in TS for example)

**Review by slinkydeveloper — APPROVED** [review]
This looks so much better now!

File: facilitators.go
Comment (jackkleeman) [reply facilitators.go:1]: meh idk what else to call it :D

**Pull request merged**